pkg/aws/eni/routing/routing.go

```diff
@@ -122,6 +129,73 @@ func Install(ip net.IP, info *RoutingInfo, mtu int, masq bool) error {
 	return nil
 }
 
+// Delete removes the ingress and egress rules that control traffic for
+// endpoints. Note that the routes within these rules are not deleted as they
+// can be reused when another endpoint is created on the same node. The reason
+// for this is that each ENI device has an index called ifindex which is used
+// as the table ID when these rules are created. The routes are created inside
+// a table with this ID, and because this table ID is tied to the ENI device,
+// it's stable to rely on and therefore can be reused.
+func Delete(ip net.IP) error {
+	ipWithMask := net.IPNet{
+		IP:   ip,
+		Mask: net.CIDRMask(32, 32),
+	}
+
+	scopedLog := log.WithFields(logrus.Fields{
+		"ip": ipWithMask,
```

Comment:
It logs how `%v` would make it look which is `{1.1.1.1 ffffffff}`, for example. It's not exactly readable and it would probably be helpful to just breakout the IP and mask portion. Will address.